packages/apollo/test/cypress/e2e/features/12-TestReader.feature

```diff
@@ -1,4 +1,4 @@
-@support @runonconsole
+@support @runonconsole @runonconsole @usermanagement
```

Comment:
@uhaller  - The tag "@runonconsole" is repeated twice. We will need @regression tag.